lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
```

Comment:
@brianparry it would be easy from the workflow engine side to mark an active or inactive key. Since it's convenient with mongo, we do most things by inference from a couple fields now (state, node, etc.), but it's just as easy to be very explicit and add fields which contain the AND logic within them, and update them accordingly when we do our write queries.